internal/alert/manager.go

```diff
@@ -78,6 +93,17 @@ func (am *alertManager) handleSlackPost(alert core.Alert) error {
 	return nil
 }
 
+// handleSlackPost ... Handles posting an alert to console log
+func (am *alertManager) handleLogPost(alert core.Alert) {
+	logger := logging.WithContext(am.ctx)
+
+	logger.Error("Pessimism Alert",
```

Comment:
Wonder if a warning would be more appropriate

Reply:
Yea, i contemplated that. It essentially depends on how downstream monitoring of logs are setup. Figured usually errors are monitored and consumed, not so much warnings. But we can make the log type as an option in the future.